src/app.rs

```diff
@@ -615,6 +615,17 @@ pub fn build_app() -> App<'static, 'static> {
                          argument. Changes the usage to `fd [FLAGS/OPTIONS] --search-path <path> \
                          --search-path <path2> [<pattern>]`",
                 ),
+        )
+        .arg(
+            Arg::with_name("strip-prefix")
+                .long("strip-prefix")
+                .short("P")
```

Comment:
Can we please remove the short option? I don't think a short prefix for this will be needed frequently, so I'd rather reserve it for future use. We should also add `.hidden_short_help(true)` here, as we don't want this to show up when calling `fd -h` (which only shows the most important options).